documentation/release_notes.rst

```diff
@@ -8,6 +8,41 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6
+=============
+
+New Features
+------------
+- Added one more thread parallel backend. It is based on OpenMP 5.0. To enable the OpenMP backend, please define macro ``ONEDPL_USE_OPENMP_BACKEND`` into 1.
+- Added the range-based version of the ``reduce_by_segment`` algorithm and improved performance of the iterator-based ``reduce_by_segment`` APIs. 
+  Please note that the use of the ``reduce_by_segment`` algorithms requires C++17.
+- Added the serial-based versions of the following algorithms: ``all_of``, ``any_of``, 
+  ``none_of``, ``count``, ``count_if``, ``for_each``, ``find``, ``find_if``, ``find_if_not``.
+  For the detailed list, please refer to `Tested Standard C++ API Reference`_. 
+
+Fixed Issues
+------------
+- Fixed hangs and errors when oneDPL is used together with oneMKL in DPC++ programs.
+- Fixed possible data races in the following algorithms used with DPC++ execution policies: ``sort``, ``stable_sort``, ``partial_sort``,``nth_element``.
+- Fixed ``param_type`` API of random number distributions to satisfy C++ ``RandomNumberDistribution`` named requirement.
+  Please note that the changes may cause API/ABI backward compatibility issues in cases when ``param_type`` is explicitly used.
```

Comment:
I'd change to "... to satisfy C++ standard requirements".

Also, the warning is not very visible. Can we use `.. note::` or even `.. caution::` here? Alternatively we can move the whole note a special subsection for "Changes affecting backward compatibility". And I would change the wording to "The new definitions of param_type are not compatible with incorrect definitions in previous library versions. Recompilation is recommended for all codes that might use param_type"

Reply:
Moved the whole note to a subsection "Changes affecting backward compatibility" and changed the last sentence to "The new definitions of param_type are not compatible with incorrect definitions in previous library versions. Recompilation is recommended for all codes that might use param_type"